src/components/GetRockets.js

```diff
@@ -0,0 +1,17 @@
+const getReservation = async () => {
+  const newResponse = [];
+  await fetch('https://api.spacexdata.com/v3/rockets')
+    .then((response) => response.json())
+    .then((data) => {
+      for (let i = 0; i <= data.length - 1; i += 1) {
+        const item = {};
+        item.rocket_name = data[i].rocket_name;
+        item.description = data[i].description;
+        item.flickr_images = data[i].flickr_images;
+        newResponse.push(item);
+      }
+    });
+  return newResponse;
+};
+
+export default getReservation;
```

Comment:
- [ ] I think it'll be good to use .map here rather than using for loop, to follow JS best practices.

Reply:
using map gives me so many errors which I tried fixing but ain't working. But I will give it a try again. Thanks

Reply:
Done!